src/edyn/util/rigidbody.cpp

```diff
@@ -216,35 +219,35 @@ void clear_kinematic_velocities(entt::registry &registry) {
 }
 
 bool validate_rigidbody(entt::entity entity, entt::registry &registry) {
-    return registry.has<position, orientation, linvel, angvel>(entity);
+    return registry.all_of<position, orientation, linvel, angvel>(entity);
 }
 
 void set_rigidbody_mass(entt::registry &registry, entt::entity entity, scalar mass) {
     EDYN_ASSERT(mass > EDYN_EPSILON && mass < large_scalar);
-    EDYN_ASSERT(registry.has<dynamic_tag>(entity));
-    EDYN_ASSERT(registry.has<rigidbody_tag>(entity));
+    EDYN_ASSERT(registry.any_of<dynamic_tag>(entity));
+    EDYN_ASSERT(registry.any_of<rigidbody_tag>(entity));
     registry.replace<edyn::mass>(entity, mass);
     registry.replace<edyn::mass_inv>(entity, scalar(1.0) / mass);
     refresh<edyn::mass, edyn::mass_inv>(registry, entity);
 }
 
 void set_rigidbody_inertia(entt::registry &registry, entt::entity entity, const matrix3x3 &inertia) {
-    EDYN_ASSERT(registry.has<dynamic_tag>(entity));
-    EDYN_ASSERT(registry.has<rigidbody_tag>(entity));
+    EDYN_ASSERT(registry.any_of<dynamic_tag>(entity));
+    EDYN_ASSERT(registry.any_of<rigidbody_tag>(entity));
     auto I_inv = inverse_matrix_symmetric(inertia);
     registry.replace<edyn::inertia>(entity, inertia);
     registry.replace<edyn::inertia_inv>(entity, I_inv);
     refresh<edyn::inertia, edyn::inertia_inv>(registry, entity);
 }
 
 void set_rigidbody_friction(entt::registry &registry, entt::entity entity, scalar friction) {
-    EDYN_ASSERT(registry.has<rigidbody_tag>(entity));
+    EDYN_ASSERT(registry.any_of<rigidbody_tag>(entity));
 
     auto material_view = registry.view<material>();
     auto manifold_view = registry.view<contact_manifold>();
     auto cp_view = registry.view<contact_point>();
 
-    auto &material = material_view.get(entity);
+    auto &material = material_view.get<edyn::material>(entity);
```

Comment:
...except in this case where the `edyn::` namespace is required because the variable has the same name as the type.